### DISCUSSION TITLE
Rules you can include in future competitions

### DISCUSSION THREAD

**Discussion by VaRa-47** [cyberbotics/wrestling#93] [General]
Hi Olivier, I wanted to tell you about some new rules you could add to future games if you decide to do them. I recently saw that one participant was doing nothing but going very slowly to the left.

So, if the goal of this competition is research, like the IEEE's goal to build a football team that can beat a human football team, this competition needs to be as close to real life as possible. Yes, this game came close to real life in many ways, but it also fell short in some ways.

It would be great if you could get rid of controllers like this because it takes away from what the game is all about.
I hope you will think about this and, if you think it's right and should be done, make the appropriate changes.

**Comment by kibergrad** [comment]
I am not sure, but are you talking about BVBDort?

**Reply by VaRa-47** [reply]
Yeah

**Comment by MJavadZallaghi** [comment]
It's not appropriate to eliminate any strategy or controller, but it's good to give awards to the strategies which are more advanced, effective, and also fair.

**Comment by omichel** [comment]
The coverage rule was designed to disadvantage robots that are not moving or moving only a little bit. I believe it is sufficient to contribute to eliminate robots that do not show an interesting behavior.
If a robot is moving slowly, this may be a good strategy and we don't want to eliminated them if that turn out to be efficient.
Also, we don't want to introduce too many rules, so that the rules remain simple and understandable to people watching games.
